app/src/main/java/com/medmapper/v33001/dto/Medicine.kt

```diff
@@ -1,15 +1,16 @@
 package com.medmapper.v33001.dto
 
+import androidx.annotation.NonNull
 import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 @Entity(tableName = "Medicine")
 data class Medicine(
     @PrimaryKey(autoGenerate = true) val medID: Int,
-    @ColumnInfo(name = "name") val name: String?,
+    @NonNull @ColumnInfo(name = "name") val name: String,
     @ColumnInfo(name = "strength") val strength: String?,
-    @ColumnInfo(name = "start date") val startDate: String?,
-    @ColumnInfo(name = "prescription length") val lengthInDays: Int
+    @NonNull @ColumnInfo(name = "start date") val startDate: String,
+    @NonNull @ColumnInfo(name = "prescription length") val lengthInDays: Int
     //,@ColumnInfo() val endDate: Date = startDate
```

Comment:
Added non nulls, as the name of the medication, start date, and start day should be required values.